src/diffusers/loaders.py

```diff
@@ -887,6 +893,11 @@ def load_lora_weights(self, pretrained_model_name_or_path_or_dict: Union[str, Di
         else:
             state_dict = pretrained_model_name_or_path_or_dict
 
+        # Convert kohya-ss Style LoRA attn procs to diffusers attn procs
+        network_alpha = None
+        if any("alpha" in k for k in state_dict.keys()):
```

Comment:
Is that the "safest" way to determine whether a checkpoint is in kohya-ss style or not? Do all checkpoints have a `"alpha"` parameter?

Reply:
According to conversion code - there also possible to check for "to_out_0_lora" in unet and "out_proj_lora" in text_encoder.
Also, for safetensors there possible to check by fields in metadata, but this not available when lora saved in pickle.

Reply:
The idea of reading metadata from the safetensor seems reliable and good! Although an API is prepared for saving metadata of safetensor, ironically there is no API for loading the metadata, so we need to directly read the file, which feels a bit awkward. I posted the code and output examples here. 
https://gist.github.com/takuma104/aaedd400e21b37d1e4297fe483e90695

I think checking 'ss_network_module' == 'networks.lora' could be one way.

Reply:
I investigated the 97 LoRA files from CivitAI that I had on hand. 62 of them had metadata, and `metadata['ss_network_module'] == 'networks.lora'`. The remaining 35 did not have metadata. Metadata seems to be saved by default, but there is an option not to save it, which might be why it's not present. The precision is about 63%. It's a rather ambiguous number. Hmm.

Reply:
> The precision is about 63%.

Could you elaborate this a bit? 

So, based on your investigation, is it safe to say that the current way of determining if the checkpoint is CivitAI is our best bet? 

Cc: @patrickvonplaten